database/migrations/2023_09_18_212703_create_save_resources_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('save_resources', function (Blueprint $table) {
+            $table->id();
```

Comment:
Same here. can later be removed.

See: https://trello.com/c/Bwu2JG1K/458-ressourcenport-im-backend-hinzuf%C3%BCgen#comment-651f13a7c290023389d18a96